### PR TITLE
f-cim-14191 into Develop

### PR DESCRIPTION
**Pull request by muhammadzubair07**

### CODE REVIEW

**Comment by shakirabbas12** [comment]
- Added `customerSLa` in channel config for customer sla implementation'
- Added   `AGENT_SLA`,`CUSTOMER_SLA_STARTED`,`CUSTOMER_SLA_EXPIRED`,`CUSTOMER_SLA_STOPPED`,
- `CUSTOMER_SLA`,`HOLD_TIMER` and `NO_RESPONSE_ENTITY` for communication b/w C_manager and C_Monitor
For jira task:
http://project.expertflow.com:8080/secure/RapidBoard.jspa?rapidView=218&view=detail&selectedIssue=CIM-14005

File: pom.xml
Comment (muhammadzubair07) [inline pom.xml:1]: Please remove the SNAPSHOT suffix from the version. SNAPSHOT versions are intended for development and testing purposes. While merging into dev, base version should be used.

Comment (muhammadzubair07) [reply pom.xml:1]: Also the same base version should be added in the Readme file a well.

File: settings.xml
Comment (muhammadzubair07) [inline settings.xml:1]: This file is not needed, please remove it.

**Pull request merged**